### DISCUSSION TITLE
Unable to initialize symfony 7 project #54296

### DISCUSSION THREAD

**Discussion by maxime-dot** [symfony/symfony#54296] [Q&A]
Hello everyone, Can anybody help me ? 
![symfony](https://github.com/symfony/symfony/assets/126958298/53721ae8-f0c9-46a1-9beb-cf331f3f045f)
I am installing new symfony 7 project but I cannot complete it. I have followed official documentation during the installation. this following command which I used to install the project : **symfony new my_project_directory --version="7.0.*" --webapp.**
I use PHP 8.2.

**Comment by lyrixx** [comment]
Could you retry? I just test and it works as expected.
![image](https://github.com/symfony/symfony/assets/408368/3cea028d-1a41-487f-b246-418c4c336ac4)


---

**and** could you paste the commande you run?

Maybe you disabled all composer plugins globally?

**Comment by alexandre-le-borgne** [comment]
The command cannot find the "symfony/webapp-pack" package. 

(The webapp flag is an alias resolved by the symfony/flex composer plugin. The declaration of the alias used: https://raw.githubusercontent.com/symfony/recipes/flex/main/index.json 
More details:
http://fabien.potencier.org/symfony4-unpack-the-packs.html
)

To sum up, symfony/flex is missing. 
Is Symfony CLI up to date?
Try to re-run the command with -vvv

The command cannot find the "symfony/webapp-pack" package. 

(The webapp flag is an alias resolved by the symfony/flex composer plugin. The declaration of the alias used: https://raw.githubusercontent.com/symfony/recipes/flex/main/index.json 

More details:
http://fabien.potencier.org/symfony4-unpack-the-packs.html
)

To sum up, symfony/flex is missing. 
Is Symfony CLI up to date?
Try to run the command with -vvv flag (not documented) and --debug

`symfony new symfony-app --version="7.0.*" --webapp -vvv --debug`
